src/applications/static-pages/simple-forms/21-10210/App.js

```diff
@@ -20,25 +20,13 @@ const App = ({ formEnabled }) => {
         >
           Submit a lay witness statement online to support a claim
         </a>
-        <a
-          className="vads-c-action-link--green"
-          href="/find-forms/about-form-21-10210/"
-        >
-          Get VA Form 21-10210 to download
-        </a>
       </>
     );
   }
 
   return (
     <>
```

Comment:
you could remove these fragments `<>...</>` as well if you want, but I forgot to do it in mine as well so either way!